app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,18 @@
+class LikesController < ApplicationController
+  def create
+    @like = current_user.likes.new
+    @like.post_id = like_params
+
+    if @like.save
+      redirect_to user_post_path(current_user, @like.post)
+    else
+      render :create
+    end
+  end
+
+  private
+
+  def like_params
+    params[:post_id]
+  end
+end
```

Comment:
I was unable to like any post by any user. Kindly fix this issue. I created [this video](https://www.loom.com/share/b77521a46c694966bc92929c5ac9150c) trying to demonstrate the problem.